android/expoview/src/main/java/host/exp/exponent/nextNotifications/ScopedNotificationRequest.java

```diff
@@ -1,4 +1,4 @@
-package versioned.host.exp.exponent.modules.universal.notifications;
+package host.exp.exponent.nextNotifications;
```

Comment:
Could we please package it under `notifications`? I know it will mix with old notifications but otherwise when removing and renaming we may run into "different class" problems since it's serializable, parcelable and stored in SharedPreferences, don't we? 🤔